src/components/Map/Map.js

```diff
@@ -105,8 +106,8 @@ Map.propTypes = {
             center: PropTypes.oneOfType([
                 PropTypes.arrayOf(PropTypes.number),
                 PropTypes.shape({
-                    lng: PropTypes.number.isRequired,
-                    lat: PropTypes.number.isRequired,
+                    lng: PropTypes.func.isRequired,
+                    lat: PropTypes.func.isRequired,
```

Comment:
Could you please give some more details on why do we need `lng` and `lat` to be functions?

Reply:
@antipin because it's the shape of the 'center' object which Google returns, but I think it would be better to call them once to get proper values and then use numbers instead so I'll fix it